setup.py

```diff
@@ -94,12 +93,12 @@ def get_extra_compile_args():
 
 def get_extra_link_args():
     if sys.platform == 'win32':
-        args = ['/subsystem:windows', '/machine:I386']
+        args = ['/subsystem:windows', '/machine:x64']
```

Comment:
we should add an auto-detection of the bitness of the platform in place of having to change it by hand.